extensions/podman/packages/extension/src/extension.ts

```diff
@@ -1281,7 +1282,8 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
     extensionApi.context.setValue(START_NOW_MACHINE_INIT_SUPPORTED_KEY, isStartNowAtMachineInitSupported(version));
     extensionApi.context.setValue(USER_MODE_NETWORKING_SUPPORTED_KEY, isUserModeNetworkingSupported(version));
     extensionApi.context.setValue(PODMAN_PROVIDER_LIBKRUN_SUPPORTED_KEY, isLibkrunSupported(version));
-    const wslHypervEnabled = (await isWSLEnabled()) && (await isHyperVEnabled());
+    wslEnabled = await isWSLEnabled();
+    const wslHypervEnabled = wslEnabled && (await isHyperVEnabled());
```

Comment:
Not really a problem, since the constant is defined in the local scope, but it might lead to a problem in future. We have the constant named `wslHypervEnabled` with local scope and we have also file scope variable `wslAndHypervEnabled` defined on the line 101.

Reply:
The file scope variable is used to keep state of the context value `WSL_HYPERV_ENABLED_KEY` so to verify if we have to actually update the context or it has already updated. 
The local scope variable is just a temporary variable used to store the result of `wslEnabled && (await isHyperVEnabled())` and pass it to `updateWSLHyperVEnabledValue` . So they are two different things. I cannot replace one with the other.

So, I'm not sure to understand the concern. Do you want me to update the name of the variables or remove the local scope and call directly `updateWSLHyperVEnabledValue(wslEnabled && (await isHyperVEnabled()))` or something else?

Reply:
Just the name, nothing else.

Reply:
I have tested it in multiple combinations, wsl and hyperv, standalone hyperv, standalone wsl and in standard and elevated mode. Now all good!

Reply:
@odockal variable name updated 👍